tests/network-tests/src/fixtures/staking/BondingRestrictedFixture.ts

```diff
@@ -0,0 +1,29 @@
+import { Api } from '../../Api'
+import { assert } from 'chai'
```

Comment:
Unused import

Reply:
removed in 983ea80b58c48c209c19fea83c2b7298bd5c26ed